Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/VJBK-Jiva-replica-failure/jiva-replica-failure

```diff
@@ -0,0 +1,176 @@
+
+bash ../Openshift-EE/utils/litmus_job_runner label='app:busybox-replica-failure-jiva' job=deploy_run_test_jiva.yml
+cd ..
+bash Openshift-EE/utils/dump_cluster_state;
+bash Openshift-EE/utils/event_updater jobname:app-replica-failure-jiva $test_name jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+
+if [ "$?" != "0" ]; then
+python3 Openshift-EE/utils/result/result_update.py $job_id VJBK 4-chaos "Jiva Replica Failure" Fail $pipeline_id "$current_time" $commit_id $gittoken
+exit 1;
+fi
+
+################
+# LitmusBook 2 #
+################
+
+run_id="jiva";test_name=$(bash Openshift-EE/utils/generate_test_name testcase=volume-replica-failure metadata=${run_id})
+echo $test_name
+
+cd litmus
+cp apps/percona/chaos/openebs_volume_replica_failure/run_litmus_test.yml run_rep_fail_test_jiva.yml
+
+: << EOF
+  ---------------------------------------------------------------------------------------------------------------------
+ | specAttribute     | kind   |         default value               | test specifc value                               |
+  ---------------------------------------------------------------------------------------------------------------------|
+ | appLabel          | env    | name=percona                        | app=busybox-rep-fail-jiva                        | 
+ | appPVC            | env    | value: percona-mysql-claim          | value: openebs-busybox-rep-kill                  |
+ | appNamespace      | env    | app-percona-ns                      | app-replica-fail-jiva                            | 
+  ----------------------------------------------------------------------------------------------------------------------
+EOF
+
+sed -i -e 's/name=percona /app=busybox-rep-fail-jiva/g' \
+-e 's/value: app-percona-ns/value: app-replica-fail-jiva/g' \
+-e 's/value: percona-mysql-claim/value: app-replica-fail-jiva/g' run_rep_fail_test_jiva.yml
+
+sed -i '/command:/i \
+          - name: RUN_ID\
+            value: '"$run_id"'\
+' run_rep_fail_test_jiva.yml
+
+echo "Running the litmus test for Busybox Deployment Scaleup.."
+cat run_rep_fail_test_jiva.yml
+
+bash ../Openshift-EE/utils/litmus_job_runner label='name:openebs-volume-replica-failure' job=run_rep_fail_test_jiva.yml
+cd ..
+bash Openshift-EE/utils/dump_cluster_state;
+bash Openshift-EE/utils/event_updater jobname:app-replica-scaleup-jiva $test_name jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+
+if [ "$?" != "0" ]; then
+python3 Openshift-EE/utils/result/result_update.py $job_id VJBK 4-chaos "Jiva Replica Failure" Fail $pipeline_id "$current_time" $commit_id $gittoken
+exit 1;
+fi
+
+echo "********Deprovisioning Busybox Application*******"
+
+test_name=$(bash Openshift-EE/utils/generate_test_name testcase=busybox-deprovision-app-scale-jiva metadata="")
+echo $test_name
+
+################
+# LitmusBook 3 #
+################
+
+cd litmus
+cp apps/busybox/deployers/run_litmus_test.yml deprovision_scaleup_jiva.yml
```

Comment:
@shashank855 can you use a different file name to copy the run_litmus_test spec.